src/formatDate.js

```diff
@@ -8,7 +8,85 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // find out which separator
+  // to split the string
+  // to convert to an array of numbers
+  const pastSeparator = fromFormat[3];
+  const oldFormat = date.split(pastSeparator);
+
+  let year = '';
+  let month = '';
+  let day = '';
```

Comment:
There's no need to initialize 'year', 'month', and 'day' as empty strings since you're immediately assigning values to them within the loop. You can declare them without initialization.